sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationIdentifier.java

```diff
@@ -61,6 +59,18 @@ public String getRawId() {
         return rawId;
     }
 
+    /**
+     * Set full id of the identifier
+     * RawId is the encoded format for identifiers to store in databases or as stable keys in general.
+     *
+     * @param rawId full id of the identifier
+     * @return CommunicationIdentifier object itself
+     */
+    public CommunicationIdentifier setRawId(String rawId) {
```

Comment:
Rewatching our [ARB review](https://msit.microsoftstream.com/video/6a8b0840-98dc-ae81-f802-f1ed0eabcd96?st=1126) I believe we agreed on making `setRawId` protected in the base class and the subclasses except for `MicrosoftTeamsUserIdentifier` and `PhoneNumberIdentifier` where `setRawId` is already public.

```suggestion
    protected CommunicationIdentifier setRawId(String rawId) {
```

Reply:
done